roles/azure_infra/templates/hosts.j2

```diff
@@ -41,8 +41,13 @@ openshift_master_console_port=443
 openshift_hosted_router_replicas=3
 openshift_hosted_registry_replicas=3
 openshift_master_cluster_method=native
+{% if master_lb_private_ip is not defined %}
 openshift_master_cluster_hostname={{ rg }}.{{ location }}.cloudapp.azure.com
 openshift_master_cluster_public_hostname={{ rg }}.{{ location }}.cloudapp.azure.com
+{% else %}
+openshift_master_cluster_hostname=ocp-master-1
```

Comment:
openshift_master_cluster_hostname={{ groups['masters'][0] }}

Reply:
Changed in commit 7a94f5c.